test/common.c

```diff
@@ -151,6 +151,19 @@ int test_remove(const gchar *dirname, const gchar *filename)
 	return res;
 }
 
+int test_lstat(const gchar *dirname, const gchar *filename, GStatBuf *buf)
+{
+	g_autofree gchar *path = NULL;
+	int res;
+
+	path = g_build_filename(dirname, filename, NULL);
+	g_assert_nonnull(path);
+
+	res = g_lstat(path, buf);
```

Comment:
Would have said that we do not need `res` here, but since this is consistent with the other methods: fine for me.